catalog-info.yaml

```diff
@@ -137,3 +137,120 @@ spec:
           branch: main
           cronline: "0 0,8,16 * * * America/New_York"
           message: "Triggers pipelines 3x daily"
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
```

Comment:
I thought we were going to manage these in our own teraform repo to avoid having to do a separate checkout in a build step?

Reply:
The other Lucene pipelines are managed that way, but these ones are mostly about ES itself. And I need secrets to do the upload, so they should be attached to a repo we own.

Reply:
Mainly the one that actually builds the lucene snapshot. It's a bit confusing the way it is as the VCS information reported in Buildkite is not for Lucene, but for ES. I'd be inclined to separate out the pipeline that builds/uploads the Lucene snapshot, from the one that updates the revision in ES.

Reply:
What about a separate pipeline that only builds the snapshot, and stores it as an artifact on the build job? Then, a separate job would have to grab the artifact and upload it.

I don't super like it, but I'm proposing it for two reasons:
- We can't put Buildkite-related code in the lucene repo, so all of the logic has to live inside the top-level step definition in the pipeline. This minimizes what we'll have to shove into the yaml file.
- If the pipeline only builds the artifact, it won't require secrets, so we can disable Vault. This seems like a good idea since we'll be running code from a repo that we don't control.

And we can of course still have one pipeline that stitches them together with trigger steps.